Library/Homebrew/test/gpg_spec.rb

```diff
@@ -13,7 +13,12 @@
         shutup do
           subject.create_test_key(dir)
         end
-        expect(dir/".gnupg/secring.gpg").to exist
+
+        begin
+          expect(dir/".gnupg/pubring.kbx").to be_file
+        rescue RSpec::Expectations::ExpectationNotMetError
+          expect(dir/".gnupg/secring.gpg").to be_file
+        end
```

Comment:
Alright, I first tried to do that but got hung up on the RSpec side of things. Also, it will require a bit of thinking to figure out how to extract the version....